mithril-common/src/test/crypto_helper/cardano/kes/setup.rs

```diff
@@ -31,16 +31,12 @@ pub fn create_kes_cryptographic_material(
     let keypair = ColdKeyGenerator::create_deterministic_keypair([party_idx as u8; 32]);
     let mut dummy_buffer = [0u8; Sum6Kes::SIZE + 4];
     let mut dummy_seed = [party_idx as u8; 32];
-    let (mut kes_secret_key, kes_verification_key) =
+    let (kes_secret_key, kes_verification_key) =
         Sum6Kes::keygen(&mut dummy_buffer, &mut dummy_seed);
-    for _ in 0..kes_period {
-        kes_secret_key
-            .update()
-            .expect("KES secret key update should not fail");
-    }
```

Comment:
For my understanding, what does this change entails ?

Reply:
These updates do not have any impact on the serialized secret key, which explains why they have been removed.